<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeakFilterDlg</class>
 <widget class="QDialog" name="PeakFilterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>膨胀/腐蚀次数</string>
   </property>
   <widget class="QPushButton" name="btnNumAdd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNumSub">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNum">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>35</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>腐蚀膨胀模式</string>
   </property>
   <widget class="QRadioButton" name="radioButtonErosion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>腐蚀</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDilate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>膨胀</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
